docs/core-concepts/validation-suites.mdx

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_label: 'Validation Suites'
+---
+
+# Validation Suites
+
+**Validation Suites** help ensure your game is functioning as intended by verifying that specific criteria are met
+over the course of a [Gameplay Session](gameplay-sessions/gameplay-sessions-getting-started).
+A Validation Suite is composed of a collection of [Scenarios](#validation-scenarios)
+and [Rules](#validation-rules). Validation Suites are managed through a dashboard in the
+[Regression Games web interface](https://play.regression.gg/validations).
+
+## Creating a Validation Suite
+
+Create a Validation Suite from the [Validation Suites dashboard](https://play.regression.gg/validations)
+by clicking the **Create Suite** button at the top of the page.
+Enter a name and optional description for your Suite, and choose a Base Gameplay Session.
+This Gameplay Session will be used to create your Scenarios,
+which can then be run against other Gameplay Sessions in the future for regression testing.
+
+![Create a Validation Suite](img/validation-suites/create-validation-suite.gif)
+
+## Validation Scenarios
+
+A **Validation Scenario** defines a functional test case by asserting that specific actions are taken during
+a Gameplay Session and that they produce expected changes in the game state.
+Steps can assert that a game scene is loaded, wait for mouse and keyboard inputs, check for the existence
+of specific game objects, and much more.
+
+Validation Scenarios execute in the same order as they appear in the Validation Suite dashboard, but order does not impact how Scenarios run.
+Each Scenario starts at the first tick of the Gameplay Session, and a Suite continues to run remaining Scenarios when one fails.
+
+### Creating a Validation Scenario
+
+Open your Suite in the [Validation Suites dashboard](https://play.regression.gg/validations) and click the **Create Scenario** button.
+This opens the **Scenario Builder**, where you can view game state data and a screenshot for each tick in the Gameplay Session,
+and use this information to build steps for your Scenario.
+
+![Screenshot of a scenario](img/validation-suites/scenario-example.png)
+
+### Validation Steps
+
+Validation Steps execute in the same order as they are defined in the Scenario Builder, from the top down.
+When a step fails, the Scenario fails and ceases execution.
+
+Many steps progress through the Gameplay Session until a tick satisfies their criteria.
+These steps are prefixed with the words **Wait for**, and they tell the next step in the Scenario to execute from that tick.
+In this way, steps run sequentially through the Session and can be used to establish cause-and-effect relationships between game state changes.
+If a "Wait for" step reaches the end of the Gameplay Session without encountering a tick that satisfies its criteria, then that step will fail.
```

Comment:
I find myself nodding along saying 'yees yeees' but not really getting it. A worked out running example might be helpful, here or at the end.

Reply:
Let me know whether the example scenario helps make sense of this text, or if I need to rework this to give the user something more concrete to hook into